python/gui/qgscolorrampbutton.sip

```diff
@@ -1,220 +1,271 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/gui/qgscolorrampbutton.h                                         *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
+
+
 
-/** \ingroup gui
- * \class QgsColorRampButton
- * A cross platform button subclass for selecting color ramps. Will open color ramp dialogs when clicked.
- * Offers live updates to button from color ramp dialog. An attached drop down menu allows for access to
- * saved color ramps, as well as option to invert the current color ramp and create new ramps.
- * \note Added in version 3.0
- */
 
```

Comment:
Yep, you are right. Wondering if sipify.pl should deal with that extra blank line scenario.